BrowserSelect/Program.cs

```diff
@@ -12,7 +12,7 @@ namespace BrowserSelect
 {
     static class Program
     {
-        public static string url = "http://google.com/";
+        public static string url = "https://www.golem.de/";
```

Comment:
any particular reason for changing this?

Reply:
damn, sorry, forgot about this. Was trying to figure out your nice project and tested quite a few different domains here - just forgot to refert it back,